components/FriendPagePreview.js

```diff
@@ -44,6 +45,17 @@ const StyledInfoList = styled.div`
   font-size: 1rem;
 `;
 
+const StyledButtonContact = styled.button`
+  background-color: white;
+  margin: 0rem 1rem 0rem 1rem;
+  padding: 0rem 1rem 0rem 1rem;
```

Comment:
`margin: 0 1rem` and `padding: 0 1rem` will do the same. If the unit value is `0` no unit is needed. :D